From 25559c59c438afe7f1a5fd7465f62a4ab87d87fc Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Wed, 8 Mar 2023 15:33:59 +0100
Subject: [PATCH 0390/1141] arm64: defconfig: add typec support in addons
 fragment

Add Type-C UCSI config flags in addons fragment.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I819c496a795fdb14b952c99a6afb3759dbea439b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/316713
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Pankaj DEV <pankaj.dev@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 9f3882277fe5..672498f89b8a 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -89,7 +89,9 @@ CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
 CONFIG_TOUCHSCREEN_GOODIX=y
 CONFIG_TOUCHSCREEN_ILI210X=y
+CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
+CONFIG_UCSI_STM32G0=m
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_STM32_USBH=y
 CONFIG_V4L_TEST_DRIVERS=y
-- 
2.39.2

